pandas/tests/test_categorical.py

```diff
@@ -797,6 +797,96 @@ def test_set_categories_inplace(self):
         cat.set_categories(['a', 'b', 'c', 'd'], inplace=True)
         tm.assert_index_equal(cat.categories, pd.Index(['a', 'b', 'c', 'd']))
 
+    @pytest.mark.parametrize(
```

Comment:
can you add an xfail with datetime64[D], something like this
```
"timedelta64[h]", pytest.param('datetime64[D]', marks=pytest.mark.xfail(
                         reason='write the issue number'))
```

Reply:
yup, just added it